src/util/camera.cpp

```diff
@@ -139,6 +139,19 @@ float Camera::maxFov() const {
     return _maxFov;
 }
 
+const glm::dmat4& Camera::combinedViewMatrixNoScale() const {
```

Comment:
I'm unsure whether this method is necessary. Right now it seems to only be used by `computeScreenSpaceData`, which I'm not sure actually needs it. Maybe it's a good method to have anyways... Thoughts?

Reply:
I will remove this for now, I could not spot any difference with using the regular combinedViewMatrix.